agent/consul/state/event.go

```diff
@@ -49,6 +49,7 @@ func (e *EventPublisher) Commit() {
 				continue
 			}
 
+			// todo(kyhavlov): handle this better
```

Comment:
We should add this to the project where we are tracking issues. I don't think this is OK for release - missing events is not great!